src/components/manager/runs/RunLastContent.js

```diff
@@ -0,0 +1,77 @@
+import React, { useState, useEffect } from "react";
+import { Table } from "antd";
+import { useSelector } from "react-redux";
+
+const LastIngestionTable = () => {
+    const runs = useSelector((state) => state.runs.items);
+
+    const [ingestions, setIngestions] = useState([]);
+
+    const processIngestions = (data) => {
+        const latestIngestions = data.reduce((acc, obj) => {
+            if (obj.state === "COMPLETE") {
+                const dataType = obj.details.request.tags.workflow_metadata.data_type;
+                const tableId = obj.details.request.tags.table_id;
+                // Skip the process if the tableId is undefined
+                if (tableId === undefined) {
+                    return acc;
+                }
+                const fileNameKey = Object.keys(obj.details.request.workflow_params)[0];
+                const filePath = obj.details.request.workflow_params[fileNameKey];
+                const fileName = filePath.split("/").pop();
+                const date = obj.details.run_log.end_time.split("T")[0];
+
+                const currentIngestion = { date, dataType, tableId, ingestedFiles: 1, fileNames: [fileName] };
+
+                const existingIngestion = acc.find((elem) => elem.dataType === dataType);
```

Comment:
based on this, i'd say that using an object would be a better accumulator than a list - you could set keys by `dataType` and then use `Object.values` with a sort by date to put them into the React state later